core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -901,8 +907,8 @@ public void send(TabletServerMutations<Mutation> tsm)
             span.end();
           }
         } catch (IOException e) {
-          if (log.isTraceEnabled()) {
-            log.trace("failed to send mutations to {} : {}", location, e.getMessage());
+          if (log.isDebugEnabled()) {
+            log.debug("failed to send mutations to {} : {}", location, e.getMessage());
           }
```

Comment:
The guarding based on the log level is unnecessary.

```suggestion
          log.debug("failed to send mutations to {} : {}", location, e.getMessage());
```

Reply:
changed in a513d75